Client/Application/LeanplumIntegration.swift

```diff
@@ -187,6 +184,13 @@ class LeanplumIntegration {
         return !UIApplication.shared.canOpenURL(klar)
     }
 
+    func mailtoIsDefault() -> Bool {
+        if let mailtoScheme = self.profile?.prefs.stringForKey(PrefsKeys.KeyMailToOption), mailtoScheme != "mailto:" {
+            return false
+        }
+        return true
```

Comment:
Why not `return self.profile?.prefs.stringForKey(PrefsKeys.KeyMailToOption) == "mailto:"`